api/test/jest.setup.js

```diff
@@ -1,4 +1,27 @@
 global.beforeAll(() => {
   global.API_BASE = 'http://localhost:8081/api/v2';
-  global.HUMAN_GEM_VERSION = '1_10_0';
+  let model = readModelInfo();
+  global.FRUITFLY_GEM_VERSION = model['Fruitfly-GEM'];
+  global.HUMAN_GEM_VERSION = model['Human-GEM'];
+  global.MOUSE_GEM_VERSION = model['Mouse-GEM'];
+  global.RAT_GEM_VERSION = model['Rat-GEM'];
+  global.WORM_GEM_VERSION = model['Worm-GEM'];
+  global.YEAST_GEM_VERSION = model['Yeast-GEM'];
+  global.ZEBRAFISH_GEM_VERSION = model['Zebrafish-GEM'];
+  global.COMPONENTS = [
+    'metabolite',
+    'gene',
+    'reaction',
+    'subsystem',
+    'compartment',
+  ];
 });
+
+function readModelInfo() {
+  let json = require('../src/data/integratedModels.json');
+  let result = {};
```

Comment:
Really minor detail here. `const` should be used instead of `let` when the values are not reassigned. Note that for the case of `result`, the object does can have its properties modified, a `let` should only be used if it is reassigned (`result = foo`). More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

```suggestion
  const json = require('../src/data/integratedModels.json');
  const result = {};
```

Reply:
@MalinAhlberg I'm committing this, writing this comment so you can trace it